src/coreclr/jit/llvm.cpp

```diff
@@ -2339,6 +2430,13 @@ void Llvm::lowerToShadowStack()
 
                 CurrentRange().InsertBefore(node, retAddressLocal, storeNode);
             }
+            if (node->OperIsLocalAddr())
+            {
+                GenTreeLclVarCommon* localAddrNode  = node->AsLclVarCommon();
+                LclVarDsc* localVarDsc = _compiler->lvaGetDesc(localAddrNode->GetLclNum());
+
+                localVarDsc->lvHasLocalAddr = 1; // TODO-LLVM: GT_LCL_FLD_ADDR will also get set to 1 here, is that a problem?
```

Comment:
```suggestion
                // Indicates that this local is to live on the LLVM frame, and will not participate in SSA.
                _compiler->lvaGetDesc(node->AsLclVarCommon())->lvHasLocalAddr = 1;
```
(`LCL_FLD_ADDR` locals should also get it of course)